### DISCUSSION TITLE
Audio effect problem when NUM_CHANNELS > 1

### DISCUSSION THREAD

**Discussion by jeremyhipp** [PlummersSoftwareLLC/NightDriverStrip#163] [Q&A] [answered]
I have used an M5StickC-Plus with several different configurations of LEDs and audio, and it has always worked fine. However, I have always had one channel of LEDs (#define NUM_CHANNELS 1).

I have now tried connecting a second channel, and the audio stops processing. The VU and spectrum bars stop processing on the display, and the effects do not function.

The debug messages show the VU is always 0:
(I) (loop)(C1) Mem: 242008 LED FPS: 62, LED Watt: 7, LED Brite: 100%, Audio FPS: 40, Serial FPS: 0, PeakVU: 0.00, MinVU: 0.00, VURatio: 0.00, CPU: 42%, 00%, FreeDraw: 0.003483

Am I doing something wrong?

**Comment by davepl** [comment, marked as answer]
Sounds like a bug!  Pls fix! :-)

Any chance it’s a pin collision, like the second channel is using the audio pin or something wonky?  

Some older code assumes Channel 0 when it draws, but Channel 0 should still work.  Not sure why a second channel would bork it, offhand!

_ Dave

**Reply by jeremyhipp** [reply]
Oh shoot... I didn't realize G0 was used for the mic on the M5 since it was routed out to the 8-pin connector.
Nevermind, it's working now.
